working-groups/design-and-ux.md

```diff
@@ -2,28 +2,78 @@
 layout: text-page
 title: Design & UX
 slug: design-and-ux
+
 ---
 
-**Status**: Active<br>
-**Bottom liner**: Georgia Bullen<br>
-**Working Group members**: Memo Esparza, Perrie Ojemeh, Eriol Fox, Bernard Tyers, Justin W. Flory, Benjamin Nickolls, Richard Littauer <br>
-**How to get involved**: See below.<br>
+# About the squad
+
+* **Status**:
+  Active
+* **Bottom-liner**:
+  _rotates monthly!_
+* **Working Group members** (alphabetically):
+  Benjamin Nickolls,
+  Bernard Tyers,
+  Eriol Fox,
+  Georgia Bullen
+  [Justin W. Flory](https://jwf.io),
```

Comment:
Seems odd to just have your website. Should we add ours? What if we have multiple websites?

Reply:
@RichardLitt Sure! I think anyone should be able to add a link for themselves if they wish. Before this merges, I could collect URLs from folks in Slack so others don't have to send their own patch if they don't want to.

Reply:
> Before this merges, I could collect URLs from folks in Slack so others don't have to send their own patch if they don't want to.

That sounds great to me. Mine is burntfen.com. Eriol's is https://erioldoesdesign.com/.

Reply:
@Erioldoesdesign Just an FYI to this comment^^, will add your personal website linked to your name on the WG profile! 👍🏻

Reply:
Resolved in commit 6b58deb4f49b618d1dcee1e607356df80deb6c9b.